tests/App/Model/ModelManager.php

```diff
@@ -55,6 +55,8 @@ public function findOneBy(string $class, array $criteria = []): ?object
 
     /**
      * @return Foo|null
+     *
+     * @psalm-suppress ImplementedReturnTypeMismatch
```

Comment:
The return phpdoc is wrong imo

Reply:
```
    public function find(string $class, $id): ?object
    {
        return $this->repository->byId($id);
    }
```
Since the repository is a Foo repository, it always return Foo or null.

I had some trouble with this issue since the modelManager is suppose to return a value corresponding to the class, but is not doing this job in the tests...

Reply:
But did you try to remove the phpdoc? the interface already have that

Reply:
If I remove the Phpdoc, I end with another error, from phpstan maybe.

Reply:
Without the phpdoc I get 
```
phpstan: Method Sonata\AdminBundle\Tests\App\Model\ModelManager::find() should return T of object|null but returns Sonata\AdminBundle\Tests\App\Model\Foo|null.
```
```
psalm: InvalidReturnStatement: The inferred type 'Sonata\AdminBundle\Tests\App\Model\Foo|null' does not match the declared return type '(T:fn-sonata\adminbundle\model\modelmanagerinterface::find as object)|null' for Sonata\AdminBundle\Tests\App\Model\ModelManager::find
```